.github/workflows/sql-odbc-main.yml

```diff
@@ -61,19 +67,19 @@ jobs:
     #    cp ./bin64/*.log test-output
     - name: upload-build
       if: success()
-      uses: actions/upload-artifact@v1
+      uses: actions/upload-artifact@v3
       with:
         name: mac64-build
         path: sql-odbc/build-output
     - name: upload-installer
       if: success()
-      uses: actions/upload-artifact@v1
+      uses: actions/upload-artifact@v3
       with:
         name: mac64-installer
         path: sql-odbc/installer
     #- name: upload-test-results
```

Comment:
any idea why this code is commented out?

Reply:
ODBC tests don't pass and/or work properly. This was commented a while ago, I still hope to fix tests and uncomment it somewhen.
It is applicable for the rest of the comment there

Reply:
Thanks.  That helps.  Could you mention that in case someone sees these comments and wants to remove them?